# Stubs for svn.exception (Python 3)
#
# NOTE: This dynamically typed stub was automatically generated by stubgen.

class SvnException(Exception): ...
